composer.json

```diff
@@ -25,7 +25,7 @@
         }
     },
     "require": {
-        "php": "^5.6 || ^7.0",
+        "php": "^5.6 || ^7.0 || ^8.0",
```

Comment:
Maybe if we use `--ignore-platform-reqs` then don't need to allow this version in composer?